app/models/katello/concerns/location_extensions.rb

```diff
@@ -8,6 +8,7 @@ module LocationExtensions
         before_create :set_katello_default
         before_save :refute_katello_default_changed
         before_destroy :deletable?
+        attr_accessible :katello_default
```

Comment:
Actually, it looks like the code did work. In the job you linked to, the db seed completely successfully. However, it looks like it fails on this line: https://github.com/Katello/katello/pull/5539/files#diff-816d256f98117950a20b7ae031224817R71.